blocks/library/paragraph/index.js

```diff
@@ -140,10 +140,7 @@ class ParagraphBlock extends Component {
 				</InspectorControls>
 			),
 			<div key="editable" ref={ this.bindRef }>
-				<Autocomplete completers={ [
-					blockAutocompleter( { onReplace } ),
-					userAutocompleter(),
-				] }>
+				<Autocomplete getBlockType={ getBlockType } onReplace={ onReplace }>
```

Comment:
We might want to iterate on `getBlockType` passed as prop. Maybe it should be a part of the autocompleter object. Not sure.

Reply:
This will go away when I make the update for BlockEdit to provide autocompleters via context. There will be no need to find the containing block to provide autocompletion context because that will be provided naturally via React context.